### DISCUSSION TITLE
OON Allowed-Amounts File Where does 20 Claim Threshold Apply

### DISCUSSION THREAD

**Discussion by APWU-GRoberts** [CMSgov/price-transparency-guide#181] [Q&A]
We are having some internal debate about where the 20 claim threshold applies for the data to be reported.  Interpretation of the Transparency Act itself vs the sample file has lead to this confusion.  

The sample file appears to group first by Billing Code, then TIN, Service Code, Allowed Amount, Billed Amount, and NPIs.
The Transparency Act refers to "Particular Provider".  Which we have interpreted to mean NPI.

The debate is as follows:

- Is the threshold at the service level?  The total number of times the service has been billed is >= 20. regardless of the TIN, Location, or NPI
- Is it at the TIN level?  The TIN has billed the service at least 20 times.  Example: TIN 888-00-1234 bills 99214 25 times.  TIN 888-00-1235 bills 99214 5 times.  TIN 888-00-1234 would be reported and TIN 888-00-1235 would NOT be reported.
- Is it at the TIN and Location?  The TIN has billed the service from the same location at least 20 times.  TIN 888-00-1234 Location 01 billed 99214 27 times. TIN 888-000-1234 Location 02 billed 99214 7 times.  Do both locations get reported? or only location 01.
- Is it at the individual provider level? (NPI)  TIN 888-00-1234 NPI 1111111111 billed 99214 27 times.  TIN 888-00-1234 NPI 1111111112 billed 99214 7 times.  Do both NPIs get reported? Or only the NPI that billed the service more than 20 times?

Thank you in advance for any guidance and clarification you can provide.

I am attaching a workbook of several different scenarios we have come up with attempting to come to a conclusion.
[OON MRF Extract Scenarios.xlsx](https://github.com/CMSgov/price-transparency-guide/files/7022744/OON.MRF.Extract.Scenarios.xlsx)

**Comment by skochin** [comment]
Our interpretation so far is that the combo of billing code, TIN, Service Code, and NPI must occur 20 times to meet the threshold, so it is at the provider level.  I am interested to hear others' thoughts or an official clarification, though.

**Reply by CKreklow** [reply]
This has been my interpretation as well.  Given the reasoning in the final rule around privacy, it would seem that the threshold must apply at the most granular level.

**Reply by ericsilvertx** [reply]
Going at the provider level will surely reduce the size of the file and the number of rows for sure.  Quick question, are we further limiting this by group and health plan as well

**Reply by ericsilvertx** [reply]
The allowed_amount is each unique allowed amount, reflected as a dollar amount, that a plan or issuer paid for a covered item or service furnished by an out-of-network provider during the 90-day time period that begins 180 days prior to the publication date of the machine-readable file. To protect patient privacy, a plan or issuer must not provide out-of-network allowed amount data for a particular provider and a particular item or service when compliance would require the plan or issuer to report out-of-network allowed amounts paid to a particular provider in connection with fewer than 20 different claims for payment. Issuers, service providers, or other parties with which the plan or issuer has contracted may aggregate out-of-network allowed amounts for more than one plan or insurance policy or contract. If information is aggregated, the 20 minimum claims threshold applies at the plan or issuer level.

**Reply by jonberkery-work** [reply]
Yes, @ericsilvertx - my company has interpreted the rule to go down to the group health plan \ plan id level for the allowed amounts file.  Therefore it is even more granular then what @skochin first described ..... 20 claims, in the timeframe, **under that group health plan \ plan id** - that have the same billing code, TIN, Service Code, and NPI.  This makes sense because this allowed amounts file is produced at the group health plan \ plan id level.

**Reply by tdfow** [reply]
Our interpretation aligns with the same as @jonberkery-work stated

**Reply by shaselton-usds** [reply]
The 20 claim threshold is counted at the billing code level for a plan (or aggregated plan reporting). There are ways to display information related to the item or service's billing code such as different allowed amounts, billed amounts, TIN information, etc.

**Reply by skochin** [reply]
@shaselton-usds your response seems to contradict the interpretation that this group was using, specifically that the 20 claim threshold applies at a provider level (TIN & NPI, in addition to plan and billing code).  Is the requirement that any billing code that occurs within the time period at least 20 times (whether all to a single provider or to many distinct providers) is reported in the file?

**Reply by tdfow** [reply]
From page 271 of TiC:  "The final rules define billed charges as total charges for an item or service billed to a plan or issuer by a provider. Plans and issuers are required to publicly disclose billed charges associated with each unique allowed amount that would be required under the final rules. The final rules further clarify that plans and issuers must report each unique combination of allowed amounts and billed charges **for each out-of-network provider**, and their associated Place of Service Code, provider NPI, and provider TIN. For example, an out-of-network provider (**under a single NPI, TIN, and Place of Service Code)** submits 25 claims (or any other number of claims to meet the 20 unique claim threshold requirement discussed in more detail later in this preamble) to a plan or issuer for the service Y. The 25 claims have three179 different billed charges ($100, $150 and $200) and two different allowed amounts ($50 and $150) for item Y."

**Reply by skochin** [reply]
@tdfow this is the example we had relied upon as well.  However the actual rule is less clear:  (b)(1)(ii)(C) "Unique out-of-network allowed amounts and billed charges with respect to covered items or services furnished by out-of-network providers during the 90-day time period that begins 180 days prior to the publication date of the machine-readable file (except that a group health plan or health insurance issuer must omit such data in relation to a particular item or service and provider when compliance with this paragraph (b)(1)(ii)(C) would require the plan or issuer to report payment of out-of-network allowed amounts in connection with fewer than **20 different claims for payments under a single plan or coverage**). Consistent with paragraph (c)(3) of this section, nothing in this paragraph (b)(1)(ii)(C) requires the disclosure of information that would violate any applicable health information privacy law. Each unique out-of-network allowed amount must be: (1) Reflected as a dollar amount, with respect to each covered item or service that is furnished by an out-of-network provider; and (2) Associated with the NPI, TIN, and Place of Service Code for each out-of-network provider."

**Reply by tdfow** [reply]
The Out of Network Object is based on Billing Code, while the Allowed Amount Array will report the payment information (array) based on the Provider TIN data where that TIN must have at least 20 unique claims for that Billing Code within the 90 day time frame from 180 days prior to publication date.  The privacy rule is at this level to ensure privacy, as if there is not at least 1 TIN with 20 or more claims then the Billing Code could not be published.  If there are multiple TINs with at least 20 claims, then there will be 'x' arrays of Allowed Amount objects when each TIN would report the unique allowed/billed charges.  Isn't that what the 'omit' reference is referring---as you couldn't publish the Out of Network Object for a Billing Code unless at least one TIN has 20 claims for that item/service?

**Reply by skochin** [reply]
This has also been our approach so far, in fact, we are not writing out the records unless the same NPI / TIN / Place of Service combo has at least 20 claims for the billing code within the time period, in line with the discussion point @tdfow references above.  The comments by @shaselton-usds and technical clarification on the  CMS site seem to only take into account the billing code and plan, so we'd like further clarification on that point.

**Reply by RLTx1391** [reply]
Any response on these questions?

**Reply by skochin** [reply]
I have not seen any clarification on this question.  @shaselton-usds can you address this?

**Comment by mmorri05** [comment]
And "billed charge" is just along for the ride and not to be used in counting service occurrences. Correct?

If I look at it from the bottom-up...
Include all NPIs:
--For (and grouped by) each distinct billed charge, when there is a count of more than 20 instances for the combination of:
----A given allowed amount:
------At a specific service code:
--------For a specific TIN:
----------Billing a specific billing code:
------------For a specific plan-id  

Edit:  I don't really see why billed charge is part of the mix

**Reply by shaselton-usds** [reply]
Billed charges isn't needed in the 20 claims count threshold. Only the billing code. Take a peek here: [After we compile all our allowed amounts and billed charges for the Allowed Amount file, how do we adjust the file to make sure we have taken into account the 20-claim threshold?](https://www.cms.gov/healthplan-price-transparency/resources/technical-clarification)

**Comment by ericsilvertx** [comment]
And that is the key looking at it from bottom up.  This makes more sense to
me.
Sincerely yours,

*_______________________________________________________*
*Eric Silver*


*(972) 999-7876*
*BTW you will have a more rapid response by texting 972-999-7876*
If you need to reach me on matters that needs attention PLEASE call me at

**Comment by shaselton-usds** [comment]
@tdflow @RLTx1391 @skochin @jonberkery-work 

Just following up on this. The 20 claim count, as noted above is based on the plan-id and billing code. Not the unique combination of TIN/NPI/billing code.

For example:
Plan A with 21 claims for service X across two TINs and multiple NPIs for two different unique allowed amounts would report the unique allowed amounts for the service even though neither TIN/NPI combination would reach 20 by itself. Service X under Plan A met the 20 count. The privacy is still intact because there is no way to identify how many claims came from what plan (if aggregated reporting happens) nor from which NPI/TIN

**Reply by gittheta** [reply]
@shaselton-usds 

We understand the clarification above and in #275 and in [CMS technical clarification question #23](https://www.cms.gov/healthplan-price-transparency/resources/technical-clarification), are all saying that the 20 claim threshold is determined using only plan-id and billing code, but we'd like to confirm that this guidance does not conflict w/ text we are seeing in the actual rule and AA schema page. See below from https://github.com/CMSgov/price-transparency-guide/tree/master/schemas/allowed-amounts:

>To protect patient privacy, a plan or issuer **must not provide out-of-network allowed amount data for a particular provider and a particular item or service when** compliance would require the plan or issuer to **report out-of-network allowed amounts paid to a particular provider in connection with fewer than 20 different claims for payment**. Issuers, service providers, or other parties with which the plan or issuer has contracted may aggregate out-of-network allowed amounts for more than one plan or insurance policy or contract. If information is aggregated, the 20 minimum claims threshold applies at the plan or issuer level.

When we see **paid to a particular provider in connection with fewer than 20 different claims for payment** our immediate thought is this is saying if issuer paid less than 20 different claims to a provider for a billing code, then we can't report allowed amounts for that particular provider. Furthermore, we can only report amounts for a billing code at all if there is at least one provider that had at least 20 different claims for that billing code. 

What all the guidance seems to be saying, though, is that as long as the claims paid to a particular provider help contribute to 20+ different claims issuer paid for a billing code across all OON providers for a given plan-id, then we should be reporting allowed amounts paid to all those providers that helped make up the 20+ different claims.

Are we understanding this properly?

**Comment by Paul-Sousa** [comment]
Our interpretation of this is the threshold criteria is dependent on whether the Allowed Amount file is for a single plan (ex. PPO Basic Plan $1000) or an aggregate of multiple plans (ex. PPO Basic Plan $1000 + PPO Basic Plan $2000 + PPO Basic Plan $3000, etc). For the former the 20 claim rule applies at the provider level, but the latter it applies at the plan/issuer level. Hope that helps (and is accurate!).

**Reply by gittheta** [reply]
With the CMS answers to discussion questions on GitHub and CMS technical clarification, it sounds like threshold is determined at plan-id + billing code level, regardless of whether MRF is reporting one plan or many plans. The clarification that the threshold applies at plan/issuer level for aggregate reporting is to ensure that _each_ plan-id the MRF is reporting meets the threshold and that it isn't being calculated across _all_ of the plan-ids being aggregated.

For example, if File A is aggregating Plans W, X, and Y, and for billing code 1234 Plan W has 10 different claims, X has 5 different claims, and Y has 5 different claims, you can't group together all their claims and say there are 20 claims so report allowed amounts for code 1234. Plan W has to have 20+ different claims for code 1234 to include its data for that code. Same for Plans X and Y.

@shaselton-usds please confirm if this understanding is accurate or interpretation shared by @Paul-Sousa is the intended one.
